crypto/bls/blst/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package blst implements a go-wrapper around a library implementing the
```

Comment:
I suggest this package be `crypto/bls12381/blst` personally

Reply:
+1 agreed better to be specific

Reply:
done